doc/kak.1

```diff
@@ -1,251 +1,282 @@
+.Ar session_id .
+.
+.It Fl c Ar session_id
+Connect to the given session
+.Ar session_id .
+.
+.It Fl s Ar session_id
+Set the current session name to
+.Ar session_id .
+.
+.It Fl ui Ar type
+Select user interface
+.Ar type ,
+which can be
+.Em ncurses ,
+.Em dummy ,
+or
+.Em json .
+.
+.It Fl clear
+Remove sessions that were terminated in an incorrect state
+.Pq e.g. after a crash .
+.
+.It Fl ro
+Begin in
+.Em readonly mode ,
+all the buffers opened will not be written to disk.
+.
+.It Ar +line Ns Oo Ar :column Oc | Ar +:
+Specify a target
+.Pf + Ar line
+and
+.Pf : Ar column
+for the first file.
+When the plus sign is followed by only a colon, then the cursor is sent
+to the last line of the file.
+.
+.It Ar file ...
+One or more
+.Ar file(s)
+to edit.
+.El
+.
+.Sh ENVIRONMENT
+.Bl -tag -width 6n
+.It Ev KAKOUNE_POSIX_SHELL
+Overrides the POSIX shell used for
+.Em %sh{...}
+expansion, which is
+.Pa /bin/sh
+.Pq Xr sh 1
+if unset.
+.
+.It Ev KAKOUNE_CONFIG_DIR
+Overrides the location of the directory containing Kakoune user
+configuration.
+If unset,
+.Pa $XDG_CONFIG_HOME/kak
+is used.
+.
+.It Ev XDG_CONFIG_HOME
+Path to the user's configuration directory.
+If unset,
+.Pa $HOME/.config
+is used.
+.
+.It Ev XDG_RUNTIME_DIR
+Path to the user's session's sockets.
+If unset,
+.Pa $TMPDIR/kakoune
+is used.
+.El
+.
+.Sh FILES
+In the paths documented below,
+.Pa <rtdir>
+refers to the runtime directory, whose value is determined in relation
+to the path to
+.Nm Ap s
+binary location:
+.Pa <rtdir>
+=
+.Pa <path_to_kak_binary>/../share/kak .
+.
+.Pp
+If not started with the
+.Fl n
+switch,
+.Nm
+will first load
+.Pa <rtdir>/kakrc ,
+which will in turn load the following additional files:
+.
+.Bl -bullet
+.It
+If the
+.Pa $KAKOUNE_CONFIG_DIR/autoload
+directory exists, recursively load every
+.Pa *.kak
+file in it, and its sub-directories.
+.
+.It
+If it does not exist, fall back to the system-wide autoload directory in
+.Pa <rtdir>/autoload ,
+and recursively load all files in a similar way.
+.
+.It
+.Pa <rtdir>/kakrc.local ,
+if it exists; this is a user-defined system-wide configuration.
+.
+.It
+.Pa $KAKOUNE_CONFIG_DIR/kakrc ,
+if it exists; this is the user configuration.
+.
+.El
+.
+.Pp
+Consequently, if the
+.Pa $KAKOUNE_CONFIG_DIR/autoload
+directory exists, only scripts stored within that directory will be
+loaded \(em the built-in
+.Pa *.kak
+files will not be.
+.
+.Pp
+Users who still want the built-in scripts to be loaded along their own
+can create a symbolic link to
+.Pa <rtdir>/autoload
+.Pq or to individual scripts in it
+in their user-configuration directory:
+.
+.Pp
+.Dl ln -s \fI<rtdir>\fR/autoload "${XDG_CONFIG_HOME:-$HOME/.config}"/kak/autoload
+.
+.Sh EXAMPLES
+.Bl -tag -width 6n
+.It Edit a file:
+.Nm
+.Pa /path/to/file
+.
+.It Edit multiple files (multiple buffers will be created):
+.Nm
+.Pa ./file1.txt
+.Pa /path/to/file2.c
+.
+.It Insert a modeline that sets the tabstop variable at the beginning of several source code files:
```

Comment:
If the screen is too narrow for this example to fit on one line, the second line will be indented, which looks odd.

Reply:
I noticed this as well and wasn't quite sure what was preferrable-- rewriting the sentence further, or trying to work around the wrapping in some way, since I couldn't find a way to have it wrap around to the start of the `.It` line anyway. I've rewritten it further.